bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -22,16 +22,20 @@ local_setup() {
     try --max 48 --delay 5 running_nginx
 }
 
-@test 'shutdown, make a snapshot, and clear everything' {
+@test 'shutdown, make a snapshot, and run factory-reset' {
```

Comment:
My only real comment here is that I'm concerned that we have a tendency to grow existing bats tests rather than creating new ones. Shouldn't each .bats file contain a test for a relatively specific thing?

Reply:
We will need to refactor them at some point.

However, we do want to combine closely related tests into a single file to reduce the number of times Rancher Desktop needs to be started and shutdown to keep running time as low as possible. It already takes 7 hours to run 4 different configurations of BATS on my old Mac Mini, and the time will only grow as we add tests.

But if tests are just aggregated, but not really related, and there is no shared startup savings, then we may want to split them into separate files.

Reply:
A lot of the tests I write build up some context, and then work with it.

I actually work on each test in an isolated file, but then plug it into the bigger bats test, if there is one, so it has the context it needs.

Reply:
> I actually work on each test in an isolated file, but then plug it into the bigger bats test, if there is one, so it has the context it needs.

If that's not an argument for smaller BATS tests, I don't know what is. We shouldn't have to move code around to develop it.

It's just hard to understand a 180 line BATS test.

But it's not relevant right now. Let's get this merged.